community/shell/src/main/java/org/neo4j/shell/apps/Man.java

```diff
@@ -198,10 +198,9 @@ public static void printHelpString( Output out, ShellServer server, boolean list
         return availableCommands;
     }
 
-    private static synchronized String availableCommandsAsString(
-        ShellServer server )
+    private static synchronized String availableCommandsAsString( ShellServer server )
     {
-        StringBuffer commands = new StringBuffer();
+        StringBuilder commands = new StringBuilder();
```

Comment:
Maybe use stream & joining collector instead?

Reply:
talked offline about it